apis/discovery/v1alpha1/foreigncluster_types.go

```diff
@@ -29,14 +29,29 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// JoinPhase indicates the phase of a peering with a remote cluster.
+type JoinPhase string
```

Comment:
I would use PeeringPhase and drop all the "Join"-references (replacing them by Peer). What do you think @giorio94 @frisso?

Reply:
Definitely agree